go-contribution-guidelines.md

```diff
@@ -18,3 +18,29 @@ If you are new to our Go development workflow:
 - You are now free to make changes to the codebase as you please.
 - You can build the binary by running `go build ./cmd/ipfs` from the go-ipfs directory.
   - NOTE: when making changes remember to restart your daemon to ensure its running your new code.
+  
+## Imports
+
+We strive to use following convention when it comes to imports. First come stdlib imports, then local repository imports and then all other external imports. Separate them using one empty new line so they are not reordered by `go fmt` or `goimports`.
```

Comment:
"We strive to use the following convention when it comes to imports. First list stdlib imports...